src/api/npls.go

```diff
@@ -115,102 +112,3 @@ func (n *NodepoolManagerAPI) GetNodepoolLabelSets(c *gin.Context) {
 
 	c.JSON(http.StatusOK, response)
 }
-
-func (n *NodepoolManagerAPI) SetNodepoolLabelSets(c *gin.Context) {
```

Comment:
Isn't this used any more?

Reply:
it not used by UI as labels are updated in cluster update request